internal/support/scheduler/v2/application/scheduler/queue.go

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scheduler
+
+import (
+	"fmt"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/models"
+)
+
+func (m *manager) addInterval(interval models.Interval, sc *Executor) {
+	m.intervalToExecutorMap[interval.Name] = sc
+	m.executorQueue.Add(sc)
+}
+
+func (m *manager) deleteInterval(sc *Executor) {
+	delete(m.intervalToExecutorMap, sc.Interval.Name)
+	// Mark as Deleted and scheduler will remove it from the queue
+	sc.MarkedDeleted = true
+}
+
+func (m *manager) addIntervalAction(sc *Executor, action models.IntervalAction) {
+	sc.IntervalActionsMap[action.Name] = action
+	m.actionToIntervalMap[action.Name] = sc.Interval.Name
+}
+
+func (m *manager) IntervalByName(intervalName string) (models.Interval, errors.EdgeX) {
+	m.mutex.Lock()
+	defer m.mutex.Unlock()
+
+	sc, exists := m.intervalToExecutorMap[intervalName]
+	if !exists {
+		return models.Interval{},
+			errors.NewCommonEdgeX(errors.KindContractInvalid,
+				fmt.Sprintf("the executor with name %s does not exist", intervalName), nil)
```

Comment:
Change to `executor with interval name %s does not exist`

Similar change to error messages below

Reply:
Removed the redundant func IntervalByName.